benchmark/blas/blas_common.hpp

```diff
@@ -425,7 +425,7 @@ void apply_blas(const char* operation_name, std::shared_ptr<gko::Executor> exec,
         for (auto _ : ic.run()) {
             op->run();
         }
-        const auto runtime = ic.compute_average_time();
+        const auto runtime = ic.compute_time(FLAGS_timer_output);
```

Comment:
FLAGS_timer_type or might be better? Same for the other places.

Reply:
I change it to timer_method